### PR TITLE
Update advanced-logging-for-iis-client-logging.md

### PR DESCRIPTION
**Pull request by kencenerelli**
Repeated word + grammar changes

### CODE REVIEW

File: iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md [hunk 1]
```diff
@@ -110,9 +110,9 @@ The Sample Logging Client for IIS Smooth Streaming will post log messages for th
 - **Seek**. The client jumps to a new location. This event doesn't invalidate the current session or create a new one. All statistics other than the following continue to be cumulative:
 
     - **c-starttime**. The client resets to a new location using the Seek function. This field as specified above suggests the time in the video where playback started.
-    - **startupTime**. Seeking may may cause the client buffer to be filled again, so we reset this value.
+    - **startupTime**. Seeking may cause the client buffer to be filled again, so we reset this value.
     - **x-duration**. The duration is reset to **0** when **c-starttime** changes.
-- **Stop**. Client progressive download and playback stopped. However, the session (**c-playerid** GUID) isn't invalidated. Restarting playback will start from 0 offset again and at that point the following fields will continue to be cumulative:
+- **Stop**. Client progressive download and playback stopped. However, the session (**c-playerid** GUID) isn't invalidated. Restarting playback will start from 0 offset again and at that point, the following fields will continue to be cumulative:
```
Comment (terrimorton) [inline iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: There should not be a comma after "point". If anywhere it should follow the word "again".

Comment (kencenerelli) [reply iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: It could be at both places if you want a subclause. :)

Comment (lextm) [reply iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: @kencenerelli consider the English materials can be translated by a machine to other languages, attention must be paid to such. I think @terrimorton 's suggestion looks better. If the paragraph is translated to Chinese, a comma after "point" will become a problem.

Comment (kencenerelli) [reply iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: @lextm Thanks for the comment Lex. I did not consider the effect of a grammar change on translated information. 
@terrimorton Please merge with the grammatical changes you think best.

Comment (bariscaglar) [reply iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: @kencenerelli I can't edit your branch directly; could you do the change on your end? Once you do, I'll go ahead and approve the merge.

File: iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md [hunk 2]
```diff
@@ -181,12 +181,12 @@ The following is a sample log from the Sample Logging Client for IIS Smooth Stre
 
 ## Summary
 
-In this walkthrough, we reviewed the IIS Advanced Logging feature and learned how to log client data. Client logging and real-time logging can make real-time analysis of log files possible.
+In this walkthrough, we reviewed the IIS Advanced Logging feature and learned how to log client data. Client logging and real-time logging can make the real-time analysis of log files possible.
```
Comment (bariscaglar) [inline iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: I don't think this "the" is not necessary. @terrimorton ?

Comment (terrimorton) [reply iis/extensions/advanced-logging-module/advanced-logging-for-iis-client-logging.md]: Agreed, it's unneeded.

**Comment by bariscaglar** [comment]
@kencenerelli The pull request is awaiting your change. Thanks!

**Pull request merged**